src/util/checkIsOnChain.ts

```
@@ -27,7 +27,7 @@ export async function checkIsOnChain(id: number | string, postType: ProposalType
 			dataFromChain = ((await api.query.council.proposals()).toJSON() as any[]) || [];
 			return Boolean(dataFromChain.filter(([id]) => id === id).length);
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)